app/models/address.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# The address associated with a household
+class Address < ApplicationRecord
+  belongs_to :household, inverse_of: :address
+
+  validates :state,
+            length: { is: 2 },
+            on: :create
```

Comment:
Any reason to only validate this on `create`? Why not validate it all the time? We don't want someone to update this with invalid entries either.